src/components/CertificateDropZone/CertificateDropZone.tsx

```diff
@@ -14,12 +14,16 @@ interface CertificateDropzoneProps {
 export const CertificateDropZone: FunctionComponent<CertificateDropzoneProps> = (props) => {
   const { toggleQrReaderVisible } = props;
   const dispatch = useDispatch();
-  const { verificationPending, verificationStatus, retrieveCertificateByActionState } = useSelector(
+  const { verificationPending, retrieveCertificateByActionState, verificationStatus, verificationError } = useSelector(
     (state: RootState) => state.certificate
   );
 
   const isVerificationPending = verificationPending;
-  const isVerificationError = verificationStatus && !isValid(verificationStatus);
+  const isVerificationError = useCallback(() => {
+    if (verificationError) return true;
+    if (verificationStatus && !isValid(verificationStatus)) return true;
+    return false;
+  }, [verificationError, verificationStatus]);
```

Comment:
hmm.. if you want can swap to 
```suggestion
const isVerificationError = useMemo(() => {
    if (verificationError) return true;
    if (verificationStatus && !isValid(verificationStatus)) return true;
    return false;
  }, [verificationError, verificationStatus]);
```

its fine as it is now

Reply:
ok